themes/default/ProfileInfo.template.php

```diff
@@ -771,8 +771,10 @@ function template_profile_block_contact()
 					$cf_show = true;
 				}
 
+				// removed $field['name'], ': ', as it was annoying
+
 				echo '
-					<li>', $field['name'], ': ', $field['output_html'], '</li>';
+					<li>', $field['output_html'], '</li>';
```

Comment:
Did you completely remove the name of the field from the profile? (Question, just to understand if it was intended or if I missed something around :wink: )